src/Arcus.Messaging.Pumps.Abstractions/MessagePump.cs

```diff
@@ -82,63 +81,71 @@ protected virtual Task HandleReceiveExceptionAsync(Exception receiveException)
             return Task.CompletedTask;
         }
 
+        // TODO: move to extension?
```

Comment:
That's also an option, both are fine for me.

Reply:
Done that.